src/ServiceControl.Plugin.Nsb6.SagaAudit/SagaAudit.cs

```diff
@@ -13,13 +15,26 @@ public SagaAudit()
             EnableByDefault();
             DependsOn<Sagas>();
         }
-        
+
         protected override void Setup(FeatureConfigurationContext context)
         {
             context.Container.ConfigureComponent<ServiceControlBackend>(DependencyLifecycle.SingleInstance);
 
             context.Pipeline.Register<CaptureSagaStateRegistration>();
             context.Pipeline.Register<CaptureSagaResultingMessageRegistration>();
+
+            context.Container.ConfigureComponent(b => new CaptureSagaStateBehavior(context.Settings.EndpointName(), BuildSerializer(context.Settings), b.Build<ServiceControlBackend>()), DependencyLifecycle.SingleInstance);
+        }
```

Comment:
Using EndpointName instead of string, similar to core behaviors that use endpoint name